### PR TITLE
(WIP) Use Ubi custom spdk module to provide storage.

### PR DESCRIPTION
**Pull request by pykello**
bdev_ubi implements copy-on-access for SPDK, so setting up storage doesn't require the initial copy operation which can take several minutes. Thus improving the provisioning time.

### CODE REVIEW

File: rhizome/host/bin/install-liburing [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# for Ubuntu 22.04 is 2.1, so we need to install it from source.
+
+GIT_REPO_LIBURING=https://github.com/axboe/liburing.git
+liburing_dir=/usr/local/src/liburing
+
+if [[ -d $liburing_dir ]]; then
+    echo "liburing source already present, not cloning"
+else
+    mkdir -p $liburing_dir
+    git clone "${GIT_REPO_LIBURING}" "$liburing_dir"
```
Comment (enescakir) [inline rhizome/host/bin/install-liburing:13]: nit: we can clone just given tag with a single revision.
```suggestion
    git clone --depth 1 --branch liburing-2.2 "${GIT_REPO_LIBURING} "$liburing_dir"
```

Comment (fdr) [reply rhizome/host/bin/install-liburing:13]: I sort of dislike the impact on legibility, wish there was some terse notation to get a symbolic name and check the contents (or the reverse)

Comment (pykello) [reply rhizome/host/bin/install-liburing:13]: This is copied from SPDK's code, so I prefer to keep it identical to how SPDK installs liburing: https://github.com/spdk/spdk/blob/master/scripts/pkgdep/common.sh#L6

I'll add a comment.

**Comment by pykello** [comment]
superseded by https://github.com/ubicloud/ubicloud/pull/918

**Pull request closed without merging**